packages/orchestrator-ui-components/src/components/WfoBadges/WfoSubscriptionSyncStatusBadge/WfoSubscriptionSyncStatusBadge.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
```

Comment:
Add a storybook story. See other badges for an example

Reply:
Added